### ISSUE TITLE
Lack of data availability / accessibility causes less scientific evidence for ICES products 

### ISSUE THREAD

**Issue by sjurl** [ices-eg/DIG#453]
Aspect| Description
-- | --
Challenges | Lack of willingness or ability to share data with ICES can mean that no data is available for the science and advice work in the ICES community.
Action required | ​​Acceptance of FAIR in the advisory plan MOU with client organisations and members Harvesting of data from web services Formal data calls Usage of the accession system.
Ownership |  Chair of DIG

Linked to Opportunity  #578 

[legacy entry ](https://community.ices.dk/Committees/DIG/Lists/DIG%20future%20challenges%20and%20opportunities/DispForm.aspx?ID=18&ContentTypeId=0x0103007DB8903096FFDD4BA4F2198AC9382C28)
</span>
</body>
</html>

**Comment by OliverWilliamsDataManager** [comment]
Converted original issue to cover challenges only for data availability / accessibility. Did not create new issue for opportunities of more data availability / accessibility because this would be more covered by other working groups who use the data.

I deemed the impact as high because without data ICES simply could not function at all and big data gaps can lead to inaccurate or misleading science / advice.

I deemed the likelihood as medium because most ICES members do have the willingness and capability to make data available and growing technology can make this process easier. However political / financial constraints can have a big effect on member countries.

**Comment by neil-ices-dk** [comment]
related to action #529

**Comment by OliverWilliamsDataManager** [comment]
Reviewed

**Comment by sjurl** [comment]
DIG 2026 - 
At this point in time we do see a change in both data collection funding, and therefore effort, and a global uncertanty with regards to data sharing wich leads us to change the likelihood from medium to high. The Impact was allready Major and will stay Major. 

Should be highlighted at SCICOM meeting
